appcli/string_parser.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+# # -*- coding: utf-8 -*-
+
+"""
+Parse strings to other Python types.
+________________________________________________________________________________
+
+Created by brightSPARK Labs
+www.brightsparklabs.com
+"""
+
+# standard library
+from distutils.util import strtobool
+from typing import List
+
+# ------------------------------------------------------------------------------
+# CLASSES
+# ------------------------------------------------------------------------------
+
+
+class StringParser:
```

Comment:
rename this to `StringTransformer` please

Reply:
Ack, done